test/ja_serializer/phoenix_view_test.exs

```diff
@@ -130,7 +130,7 @@ defmodule JaSerializer.PhoenixViewTest do
 
   test "render conn, show.json-api, data: nil" do
     json = @view.render("show.json-api", conn: %{}, data: nil)
-    assert json['data'] == nil
+    assert json[~c"data"] == nil
```

Comment:
I think this was a typo and it should actually be `json["data"]`.

Reply:
good point, that does make sense - I should have looked more at the code instead of simply replacing it 😃